christfellowshipchurch/web-app-v2
# Setup join meeting buttons

Adds Zoom join meeting setup. I added a `handleOnClickVideoCall` in preparation for implementing amplitude and check-in.

TO TEST: 
Go to group single page and click join meeting. Zoom link should open in a new tab.

![image](https://user-images.githubusercontent.com/2528817/114617750-2a015c80-9c6e-11eb-97f5-bc12fac06c65.png)


![image](https://user-images.githubusercontent.com/2528817/114617654-063e1680-9c6e-11eb-84bd-6373360379e6.png)
